src/main/java/bio/terra/service/resourcemanagement/google/GoogleProjectService.java

```diff
@@ -59,23 +63,33 @@
                 "firebaserules.googleapis.com",
                 "storage-component.googleapis.com",
                 "storage-api.googleapis.com",
-                "cloudbilling.googleapis.com"));
+                "cloudbilling.googleapis.com",
+                "appengine.googleapis.com"));
+
+    /**
+     * This is where the Firestore database will be created.
+     */
+    private static final String DEFAULT_FS_LOCATION_ID = "us-central";
```

Comment:
I think this should be a config parameter. That is how we handle GCS bucket location right now.
Although maybe it doesn't matter, since the real fix is plumbing in user-specified location on these things.

Reply:
Ah, great call.  Will update.  I'll keep default in the name since ultimately, we may want this to be on a project/project basis